src/tools/lsp.ts

```diff
@@ -112,6 +111,37 @@ export function testGlob (pattern: string, value: string): boolean {
   return regExp.test(value)
 }
 
+const sep = process.platform === 'win32' ? '\\' : '/'
```

Comment:
can't we use path.sep ? https://nodejs.org/docs/latest/api/path.html#pathsep

Reply:
maybe, this line comes from vscode itself tho

Reply:
ok let's leave it as it is now then !